src/lib/elements/Placeholder/Placeholder.tsx

```diff
@@ -0,0 +1,42 @@
+import type { ReactNode, CSSProperties } from "react";
+import "./Placeholder.scss";
+
+import classNames from "classnames";
+
+export type PlaceholderProps = {
+  className?: string;
+  isPending?: boolean;
+  style?: CSSProperties;
+} & (
+  | {
+      text?: never;
+      children?: ReactNode;
+    }
+  | {
+      text?: string;
+      children?: never;
+    }
+);
+
+export const Placeholder = ({
+  text,
+  children,
+  className,
+  isPending = false,
+  style,
```

Comment:
```suggestion
```
Please correct me if I'm wrong, but I don't think there's actually a need to allow for custom inline styles. This isn't a pattern that we want to encourage either.

Reply:
You're right. I've pushed a commit removing this